client/app/views/recipient/tips.html

```diff
@@ -126,6 +127,7 @@
                 <span data-ng-show="sortReverse" class="fa-solid fa-caret-up"></span>
               </span>
             </span>
+            <span class="DropdownMultiselectContainer"><i class="fa-solid fa-filter"></i> <span translation-texts="dropdownDefaultText"  data-ng-dropdown-multiselect="" options="dropdownStatusData" selected-model="dropdownStatusModel" events="on_changed"  extra-settings="dropdownSettings"> </span></span>
```

Comment:
extra spaces here.

I'm going to fix these myself while integrating the pull request.